here-naksha-app-service/src/test/java/com/here/naksha/app/service/EventHandlerApiTest.java

```diff
@@ -49,15 +64,14 @@ void tc0100_testCreateEventHandler() throws Exception {
     assertThat(response)
         .hasStatus(200)
         .hasStreamIdHeader(streamId)
-        .hasJsonBody(expectedCreationResponse);
+        .hasJsonBodyFromFile("EventHandlerApi/TC0100_createEventHandler/response.json");
```

Comment:
I know this is an attempt to reduce lines of code a bit, but I would have personally continued it with existing-way to keep readability in mind that all "test data reading" happens upfront during preparation section and during assertions we only use the data that is already read earlier.
That approach is relatively a bit easier to follow. As it doesn't make significant difference so I don't have any objection on this to change it back, but let's also keep this point in mind for such code optimization. And also, I recommend not to make this change to other Tests (outside of EventHandlerApiTest).

Reply:
Sure thing, I'll adjust it :) It's not that important to me